app/controllers/calendar_controller.rb

```diff
@@ -6,7 +6,7 @@ class CalendarController < ApplicationController
 
   rescue_from Calendar::CalendarNotFound, with: :simple_404
 
-  slimmer_template "core_layout"
+  slimmer_template "gem_layout"
```

Comment:
This can also be safely removed as it inherits this from `ApplicationController`